roles/openshift_manage_node/tasks/main.yml

```diff
@@ -0,0 +1,11 @@
+- name: Handle unscheduleable node
+  delegate_to: "{{ openshift_first_master }}"
```

Comment:
I'm going to be testing out another update to install.openshift.com.  I'll try removing the delegate_to here and if that looks good I'll submit a PR.